librz/core/cmd_history.c

```diff
@@ -0,0 +1,31 @@
+RZ_IPI RzCmdStatus rz_list_or_exec_history_handler(RzCore *core, int argc, const char **argv) {
+	if (argc == 1) {
+		rz_line_hist_list();
+		return RZ_CMD_STATUS_OK;
+	}
+
+	int index = atoi(argv[1]);
+	if (index < 1) {
+		RZ_LOG_ERROR("index must be a positive number.\n");
+		return RZ_CMD_STATUS_WRONG_ARGS;
```

Comment:
Mmm maybe STATUS_ERROR even in this case? This is no different than passing a too big number.